lang_qc/models/pacbio/well.py

```diff
@@ -151,10 +151,10 @@ def from_orm(cls, mlwh_db_row: PacBioRunWellMetrics, qc_session: Session):
         if len(experiment_info):
             obj.experiment_tracking = PacBioExperiment.from_orm(experiment_info)
 
-        qced_products = get_qc_states_by_id_product_list(
-            session=qc_session, ids=[id_product], sequencing_outcomes_only=True
-        ).get(id_product)
-        if qced_products is not None:
-            obj.qc_state = qced_products[0]
+        qc_state_db = get_qc_state_for_product(
```

Comment:
Seeing this used here, it seems we have a disparity - sometimes "for_product" means you can use a product ID, other times it means an object representation.

Reply:
Yep. We have a series of historical changes and additions in `lang_qc.db.helper.qc`, which are not always consistent. This is a slightly different story. Trying to fix in the next pr